systemtests/tests/bconsole-pam/python-bareos-unittest.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
+
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+import logging
+import os
+import unittest
+
+import bareos.bsock
+import bareos.exceptions
+from   bareos.bsock.protocolversions import ProtocolVersions
+
+
+class PythonBareosBase(unittest.TestCase):
+    director_address = 'localhost'
+    director_port = 9101
```

Comment:
The port number should be dynamic. This is mandatory when running tests parallel. 
-> pstorz, arogge

Reply:
This is done below by reading the BAREOS_DIRECTOR_PORT environment variable.